examples/src/main/java/ExampleListOfRecording.java

```diff
@@ -0,0 +1,42 @@
+import com.messagebird.MessageBirdClient;
+import com.messagebird.MessageBirdService;
+import com.messagebird.MessageBirdServiceImpl;
+import com.messagebird.exceptions.GeneralException;
+import com.messagebird.exceptions.UnauthorizedException;
+import com.messagebird.objects.voicecalls.RecordingResponse;
+
+public class ExampleListOfRecording {
+    public static void main(String[] args) {
+        if (args.length < 3) {
+            System.out.println("Please specify your access key and call id and leg id and recording id example :" +
```

Comment:
System.out.println(".....recording id") . >>>> delete recoring id part, not needed for list